src/doc/src/reference/environment-variables.md

```diff
@@ -310,7 +310,7 @@ let out_dir = env::var("OUT_DIR").unwrap();
   those defined in `RUSTFLAGS`). Some examples of what these variables are:
     * `CARGO_CFG_UNIX` — Set on [unix-like platforms].
     * `CARGO_CFG_WINDOWS` — Set on [windows-like platforms].
-    * `CARGO_CFG_TARGET_FAMILY=unix` — The [target family], either `unix` or `windows`.
+    * `CARGO_CFG_TARGET_FAMILY=unix` — The [target family], for example `unix` or `windows`.
```

Comment:
I think since none of the other entries list examples, it isn't particularly important to include it here. That way, we don't need to have subtle confusion such as adding `wasm` or keeping this list up-to-date in the future. I don't recall why I wrote it this way.  These already include a single example on the left-side anyways.

```suggestion
    * `CARGO_CFG_TARGET_FAMILY=unix` — The [target family].
```